src/elements/emby-slider/emby-slider.js

```diff
@@ -190,10 +190,7 @@ function setMarker(range, valueMarker, marker, valueProgress) {
             return;
         }
 
-        let markerPos = (bubbleTrackRect.width * valueMarker / 100) - markerRect.width / 2;
-        markerPos = Math.min(Math.max(markerPos, - markerRect.width / 2), bubbleTrackRect.width - markerRect.width / 2);
-
-        marker.style.left = markerPos + 'px';
+        marker.style.left = `calc(${valueMarker}% - ${markerRect.width / 2}px)`;
```

Comment:
I don't think we can use the CSS calc function as this won't work on the older browsers we need to support

Reply:
We want to support webOS 1.2 and Tizen 2.3. Both are old WebKit which is currently approximated with Chrome 27. It's not perfect, and sometimes it's better to test in an emulator.